pytorch_lightning/accelerators/accelerator_connector.py

```diff
@@ -169,7 +243,10 @@ def use_dp(self):
 
     @property
     def use_ddp(self):
-        return self._distrib_type in (DistributedType.DDP, DistributedType.DDP_SPAWN)
+        return self._distrib_type in (
```

Comment:
This should be

```python
return self._distrib_type.uses_ddp()
```

so the logic for which ones use ddp is encapsulated in DistributedType

Reply:
We would like to do an instance check in the future on the actual plugin that was selected, to cover the case of user provided plugin

Reply:
Sounds good

When is "in the future"? If it's not going to be in the next week or so, I feel we could do this small refactor and then change it again later.

No strong feelings though.

Reply:
I will prefer to keep this small cleaning for 1.2.1, as we still have several important PR to be through.

Reply:
And this PR is mainly `re-organizing code`, not introducing new ones.

Reply:
how does this align with having the same properties in Trainer?